### DISCUSSION TITLE
12月這麼多節日！大家怎麼過？

### DISCUSSION THREAD

**Discussion by Elaine4488** [1101-GS4719-C/Python-Community-GS4719-C#3675] [General]
冬至吃湯圓、聖誕節交換禮物、新年倒數，每個都超有氣氛～期末考壓力這麼大，但這些節日又不想錯過。我通常會規劃好讀書時間，抽空和朋友慶祝一下，感覺會更有動力準備考試！那你們呢？怎麼在這個月找到讀書和過節的平衡呢？

**Comment by yue083139** [comment]
我個人的情況來說的話，其實我的期末考都分得蠻開的，平均在16-18週，所以一個禮拜也就1-2科考試，而且又很多課實際上已經沒事情，所以只要拿週末來抽空讀好該讀的範圍，還是有時間可以慶祝一下節日的（大概🤔），但天冷也不一定有動力出門就是了。

**Comment by Tchang323** [comment]
12月都跟作業、考試過哈哈

**Comment by Tchang323** [comment]
除了作業考試之外，其實還參加蠻多舞會、跟社團的期末社大，很充實的一個月~

**Comment by Mminniehuang** [comment]
因為我是所有期末考都擠在16週，時間緊湊到根本沒有任何可以放鬆的時間，所以冬至啊、聖誕夜啊，我都是跟書本和平板一起過的。不過雖然很無奈但也蠻開心終於不會錯過跨年了...
「蛤？為什麼突然說要吃湯圓」「欸？今天是聖誕夜！？」
讀到甚至根本忘記也沒有意識到今天是什麼節日的感覺，想想還是有點難過。

**Comment by Taoyi-Chiang** [comment]
完全沒有辦法想這些事情，好可憐。
我自己只有準備簡單的禮物和卡片給比較好的人，但都是用店到店，真的好好笑。

**Comment by poyen0806** [comment]
以我個人的情況來看，我12月的節日都是在考試和報告中度過的，可喜可賀，可喜可賀

每次都是身旁的人提醒，才想到有這些節日，例如：冬至、聖誕節，不知不覺就過去了...我只能跟電子書一起過

**Comment by irenesunnn** [comment]
不過也是一個方法 省錢又省力，畢竟十六周期末已經夠累了

**Comment by lichilun** [comment]
12月確實是一個充滿節日的月份，從聖誕節到冬至，還有一些地方的年終慶祝活動，這個月我的慶祝有：和家人一起過聖誕節，交換禮物、聚餐，享受溫馨的時光；參加年終聚會，和朋友一起慶祝一年的結束；參加冬至活動，吃湯圓、圍爐，享受溫暖的時光。

**Comment by yurukung** [comment]
過完聖誕節再跨年

**Comment by vu45tp6** [comment]
我覺得可以跟好友們組成一個團體，在讀書的時候互相督促增加彼此的效率。這樣在過節的時候就不會因為覺得自己還需要專注於課業上感到壓力大，也可以和朋友維繫感情。

**Comment by a414414** [comment]
12月有這麼多節日，我通常都跟課本過。12月已經接近期末了，很多考試接踵而來，壓力大到會讓人無暇慶祝節日，但也有可能是因為我平常時間管理做得不好，如果我平常就都有在讀書，一點一點累積，期中或期末應該就不會過得那麼辛苦，所以說到底還是看個人的時間管理做得如何，做得好的話就能開心地過12月的這些節日。
